### PR TITLE
[WIP] Create TF Serving notebook

### PR DESCRIPTION
**Pull request by deep-diver**
remaining work
- add more comments
- add how to build docker image for k8s

### CODE REVIEW

File: notebooks/TF_Serving.ipynb [hunk 1]
```diff
@@ -0,0 +1,785 @@
+{
```
Comment (sayakpaul) [inline notebooks/TF_Serving.ipynb:1]: <p>Maybe add a reference to the official TF Serving guides here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/ml-deployment-k8s-fastapi/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (sayakpaul) [inline notebooks/TF_Serving.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    # --enable_model_warmup for warmup(https://www.tensorflow.org/tfx/serving/saved_model_warmup)
</pre><p> </p><p>Worth adding a note on why warming up is needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/ml-deployment-k8s-fastapi/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (sayakpaul) [inline notebooks/TF_Serving.ipynb:1]: <p>I would prefer adding an assertion statement here to ensure the original model outputs match with the RestAPI and gRPC outputs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/ml-deployment-k8s-fastapi/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

**Comment by deep-diver** [comment]
@sayakpaul 

Modified the notebook based on your comment :)

**Review by sayakpaul — APPROVED** [review]

**Comment by sayakpaul** [comment]
Great! 

Feel free to add the rest of the parts. I think we could do the rest of the components via scripts as done for the ONNX model. In that case, feel free to merge this PR.

**Pull request merged**